src/Components/ServiceScene/Breakdowns/LabelValuesBreakdownScene.tsx

```diff
@@ -197,9 +261,58 @@ export class LabelValuesBreakdownScene extends SceneObjectBase<LabelValueBreakdo
     });
   }
 
+  private showErrorToast(errors: DisplayErrors) {
```

Comment:
Probably deserves a more unified place than stuffing this into the Label breakdown, but good for now.